elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/timegrains/Day.java

```diff
@@ -9,14 +9,13 @@
 import com.yahoo.elide.core.utils.coerce.converters.Serde;
 
 import java.sql.Date;
-import java.sql.Timestamp;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 
 /**
  * Time Grain class for Day.
  */
-public class Day extends Date {
+public class Day extends Date implements TimeGrainFormatter {
```

Comment:
Why is a Day an instance of TimeGrainFormatter?  Can we remove this association?

Reply:
TimeGrainFormatter has common logic that will be used by all the Grains for ISO handling. So instead of creating a  Util class with that supporting method, we went the implement/interface route.

Reply:
maybe DaySerde can implement it instead of Day.

Reply:
We are calling a static method on the interface.  Technically nothing needs to implement it.  DaySerde would make more sense though if we do.